plugins/ccxtExchange_test.go

```diff
@@ -556,24 +557,17 @@ func TestCancelOrder_Ccxt(t *testing.T) {
 }
 
 func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
-	// coinbasepro gives incorrect precision values so we do not test it here
 	testCases := []struct {
 		exchangeName       string
 		pair               *model.TradingPair
 		wantPricePrecision int8
 		wantVolPrecision   int8
 	}{
 		{
-			// disable ccxt-kraken based tests for now because of the 403 Forbidden Security check API error
-			// 	exchangeName:       "kraken",
-			// 	pair:               &model.TradingPair{Base: model.XLM, Quote: model.USD},
-			// 	wantPricePrecision: 6,
-			// 	wantVolPrecision:   8,
-			// }, {
 			exchangeName:       "binance",
 			pair:               &model.TradingPair{Base: model.XLM, Quote: model.USDT},
-			wantPricePrecision: 5,
-			wantVolPrecision:   1,
+			wantPricePrecision: 4,
+			wantVolPrecision:   4,
```

Comment:
Can you double check volume precision, I think it should be 0, not 4

Reply:
there is some logic that sets it to the price when its 0 
	volumePrecision := ccxtMarket.Precision.Amount
	if volumePrecision == 0 {
		volumePrecision = ccxtMarket.Precision.Price
	}
	
	so it ends up at 4